specification/sdk-tracing.md

```diff
@@ -260,6 +260,53 @@ high contention in a very high traffic service.
 * `maxExportBatchSize` - the maximum batch size of every export. It must be
   smaller or equal to `maxQueueSize`. The default value is `512`.
 
+##### Span watcher processor
+
+This implementation passes spans that have been started but not yet ended
+to the configured exporter.
+On each interval defined by `reportIntervalMillis`, all un-ended spans that
+have been started more than `reportIntervalMillis` ago, are exported as is.
```

Comment:
What does "as is" mean here?

Reply:
_as is_ as in _as they currently are_, i.e., their current state
that's actually redundant, however, since there's no other (future) state that can be exported anyway, so I removed it